frontend/src/modules/organization/components/edit/identity/organization-identity-add.vue

```diff
@@ -0,0 +1,196 @@
+              Add identity
+            </lf-button>
+          </lf-organization-details-identity-add-dropdown>
+        </div>
+      </div>
+      <div class="py-4 px-6 border-t border-gray-100 flex items-center justify-end gap-4">
+        <lf-button type="secondary-ghost" @click="close">
+          Cancel
+        </lf-button>
+        <lf-button
+          type="primary"
+          :loading="sending"
+          :disabled="$v.$invalid || sending"
+          @click="addIdentities()"
+        >
+          Add {{ pluralize('identity', form.length) }}
+        </lf-button>
+      </div>
+    </template>
+  </lf-modal>
+</template>
+
+<script setup lang="ts">
+import LfModal from '@/ui-kit/modal/Modal.vue';
+import { computed, reactive, ref } from 'vue';
+import LfButton from '@/ui-kit/button/Button.vue';
+import LfIcon from '@/ui-kit/icon/Icon.vue';
+import LfInput from '@/ui-kit/input/Input.vue';
+import { CrowdIntegrations } from '@/integrations/integrations-config';
+import LfCheckbox from '@/ui-kit/checkbox/Checkbox.vue';
+import Message from '@/shared/message/message';
+import pluralize from 'pluralize';
+import useVuelidate from '@vuelidate/core';
+import LfField from '@/ui-kit/field/Field.vue';
+import LfFieldMessages from '@/ui-kit/field-messages/FieldMessages.vue';
+import {
+  Organization,
+  OrganizationIdentity,
+  OrganizationIdentityType,
+} from '@/modules/organization/types/Organization';
+import { useOrganizationStore } from '@/modules/organization/store/pinia';
+import { Platform } from '@/shared/modules/platform/types/Platform';
+import LfOrganizationDetailsIdentityAddDropdown
+  from '@/modules/organization/components/details/identity/organization-details-identity-add-dropdown.vue';
+import { required } from '@vuelidate/validators';
+
+const props = defineProps<{
+  modelValue: boolean,
+  identities: Partial<OrganizationIdentity>[],
+  organization: Organization,
+}>();
+
+const emit = defineEmits<{(e: 'update:modelValue', value: boolean): void}>();
+
+const { updateOrganization } = useOrganizationStore();
+
+const sending = ref<boolean>(false);
+
+const defaultForm: OrganizationIdentity = {
+  value: '',
+  verified: true,
+  platform: Platform.CUSTOM,
+  type: OrganizationIdentityType.USERNAME,
+  sourceId: null,
+};
+
+const form = reactive<OrganizationIdentity[]>(props.identities.map((i) => ({ ...defaultForm, ...i })));
+
+const rules = computed(() => ({
+  form: form.map((item) => ({
+    value: {
+      required,
+    },
+  })),
+}));
+
+const $v = useVuelidate(rules, { form });
+
+const isModalOpen = computed<boolean>({
+  get() {
+    return props.modelValue;
+  },
+  set(value: boolean) {
+    emit('update:modelValue', value);
+  },
+});
+
+const platform = (platform: string) => CrowdIntegrations.getConfig(platform);
+
+const addIdentities = () => {
+  sending.value = true;
+  updateOrganization(props.organization.id, {
+    identities: [
+      ...props.organization.identities,
+      ...form,
+    ],
+  })
```

Comment:
For LinkedIn you should always add a prefix `company:` to the value.
Same thing for edit. Please double check that whatever frontend send has a company: prefix